File: tests/fixtures/rust/cases/src/bin/env_vars_moon.rs
use std::env;

fn main() {
    let mut vars = vec![];

    for (key, value) in env::vars() {
        if key.starts_with("MOON_")
            && !key.starts_with("MOON_TEST")
            && key != "MOON_VERSION"
            && key != "MOON_APP_LOG"
        {
            vars.push(format!("{key}={}", value.replace('\\', "/")));
        }
    }

    vars.sort();

    println!("{}", vars.join("\n"));
}

File: tests/fixtures/deno/base/envVarsMoon.ts
Object.entries(Deno.env.toObject())
	.sort((a, d) => a[0].localeCompare(d[0]))
	.forEach(([key, value]) => {
		if (
			key.startsWith('MOON_') &&
			!key.startsWith('MOON_TEST') &&
			key !== 'MOON_VERSION' &&
			key !== 'MOON_APP_LOG'
		) {
			console.log(`${key}=${value.replace(/\\/g, '/')}`);
		}
	});
